.gitignore

```diff
@@ -14,3 +14,4 @@
 mkmf.log
 tests/.fog
 gemfiles/Gemfile-edge.lock
+.idea/
```

Comment:
Could you omit this? I don't think it's strictly needed, so prefer to keep things streamlined.

Reply:
@masstamike You'll likely want to add `/.idea` to your global git ignores.  See https://help.github.com/articles/ignoring-files/

Reply:
Ah thanks @lanej! I'll remove that.